pytorch_lightning/loggers/base.py

```diff
@@ -310,7 +311,15 @@ def finalize(self, status: str) -> None:
         self.save()
 
     def close(self) -> None:
-        """Do any cleanup that is necessary to close an experiment."""
+        """Do any cleanup that is necessary to close an experiment.
+
+        .. deprecated:: v1.5     This method is deprecated in v1.5 and will be removed in v1.7.     Please use
+        `LightningLoggerBase.finalize` instead.
+        """
```

Comment:
```suggestion
        """
        .. deprecated:: v1.5
            This method is deprecated in v1.5 and will be removed in v1.7.
            Please use `LightningLoggerBase.finalize` instead.
        
        Do any cleanup that is necessary to close an experiment.
        """
```
trying to move it at the top, lets see